coverage/coverage_test.go

```diff
@@ -44,14 +48,6 @@ func TestCoverage_ResourceGroup(t *testing.T) {
 		t.Fatalf("process coverage: %+v", err)
 	}
 
-	if model.CoveredCount != 1 {
```

Comment:
I guess it's necessary to check whether the model.CoveredCound equals to tc.expectedCoveredCount? And same for the expectedTotalCount?

Reply:
please see comment below